oapen-engine/src/tasks/config.py

```diff
@@ -21,4 +21,4 @@
 
 # Update items that were modifed since X days ago
 UPDATE_DAYS_BEFORE = 30
-REFRESH_IMPORT_LIMIT = 50
+REFRESH_IMPORT_LIMIT = 0
```

Comment:
why do we need to change this?

Reply:
Probably more than 50 items that have been modified in the past week, but not an egregious amount, therefore removing limit